src/api/parser/src/utils/html/sanitize.js

```diff
@@ -50,6 +51,8 @@ module.exports = (dirty) => {
     allowedIframeHostnames: [
       'www.youtube.com',
       'player.vimeo.com',
+      'player.twitch.tv',
+      WEB_URL,
```

Comment:
Why do we need this here?  Won't this only show-up on the URL via `parant=`?  We're not actually hosting content at this URL?

Reply:
Remove this